.swiftlint.yml

```diff
@@ -1,10 +1,11 @@
-swiftlint_version: 0.54.0
+swiftlint_version: 0.58.2
```

Comment:
We cannot use `0.54.0` because it's not available as a built tool plugin, see https://github.com/SimplyDanny/SwiftLintPlugins/releases

Given that, it makes sense to use the latest available version. But even if we didn't want to:

- 0.58.0 and 0.58.1 have a bug where the Docker image reads the wrong version, so we cannot use them
- 0.55.0 does not have the `--working-directory` option which we need